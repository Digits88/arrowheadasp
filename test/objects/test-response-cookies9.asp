Test Response.Cookies, simple cookies

Response.Cookies("Hello")("First") = "One"
    <% Response.Cookies("Hello")("First")= "One" %><br>
Response.Cookies("Hello")("Second") = "Two"
    <% Response.Cookies("Hello")("Second") = "Two" %><br>
Response.Cookies("Hello").HasKeys =
    <%= Response.Cookies("Hello").HasKeys %><br>
