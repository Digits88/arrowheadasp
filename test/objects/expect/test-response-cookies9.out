Test Response.Cookies, simple cookies

Response.Cookies("Hello")("First") = "One"
    <br>
Response.Cookies("Hello")("Second") = "Two"
    <br>
Response.Cookies("Hello").HasKeys =
    True<br>
